gapic/templates/tests/unit/gapic/%name_%version/%sub/test_macros.j2

```diff
@@ -6,7 +6,7 @@
 ])
 def test_{{ method_name }}(request_type, transport: str = 'grpc'):
     client = {{ service.client_name }}(
-        credentials=ga_credentials.AnonymousCredentials(),
+        credentials=AnonymousCredentialsWithUniverseDomain(),
```

Comment:
Since validation step is a part of each RPC call, we need to pass in credentials with universe domain property to each test. Otherwise, they will complain about missing `universe_domain` property in the validation step.